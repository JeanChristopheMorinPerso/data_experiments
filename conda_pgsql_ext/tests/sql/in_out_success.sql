CREATE EXTENSION condaversion;
SELECT '1.0.0'::condaversion;
SELECT ' 1.0.0 '::condaversion;
SELECT '1.0.0'::condaversion;
SELECT ' 1.0.0 '::condaversion;
