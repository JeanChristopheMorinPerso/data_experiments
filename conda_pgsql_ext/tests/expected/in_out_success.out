CREATE EXTENSION condaversion;
ERROR:  extension "condaversion" already exists
SELECT '1.0.0'::condaversion;
 condaversion 
--------------
 1.0.0
(1 row)

SELECT ' 1.0.0 '::condaversion;
 condaversion 
--------------
  1.0.0 
(1 row)

SELECT '1.0.0'::condaversion;
 condaversion 
--------------
 1.0.0
(1 row)

SELECT ' 1.0.0 '::condaversion;
 condaversion 
--------------
  1.0.0 
(1 row)

